battery_check.py

```diff
@@ -0,0 +1,19 @@
+
+from check_main_limit import check_function
+from check_warning_limit import warning_function
+
+def battery_is_ok(temperature, soc, charge_rate,temp_min,temp_max,soc_min,soc_max,charge_rate_min,charge_rate_max):
```

Comment:
too many parameters in the method API. Data organization can be better by usage of data structures.